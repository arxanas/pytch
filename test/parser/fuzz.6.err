INVALID_TOKEN[1000] in test/parser/fuzz.6.pytch, line 1, character 4:
Error: Invalid token '!'.
   +--------------------------------+
   | test/parser/fuzz.6.pytch       |
 1 | let!f/o = print@let            |
   |    ^ Error: Invalid token '!'. |
   +--------------------------------+
INVALID_TOKEN[1000] in test/parser/fuzz.6.pytch, line 1, character 6:
Error: Invalid token '/'.
   +----------------------------------+
   | test/parser/fuzz.6.pytch         |
 1 | let!f/o = print@let              |
   |      ^ Error: Invalid token '/'. |
   +----------------------------------+
INVALID_TOKEN[1000] in test/parser/fuzz.6.pytch, line 1, character 16:
Error: Invalid token '@'.
   +--------------------------------------------+
   | test/parser/fuzz.6.pytch                   |
 1 | let!f/o = print@let                        |
   |                ^ Error: Invalid token '@'. |
   +--------------------------------------------+
EXPECTED_PATTERN[1013] in test/parser/fuzz.6.pytch, line 1, character 4:
Error: I was expecting a pattern after 'let'.
   +-----------------------------------------------------+
   | test/parser/fuzz.6.pytch                            |
 1 | let!f/o = print@let                                 |
   |    ^ Error: I was expecting a pattern after 'let'.  |
   | ^~~ Note: This is the beginning of the let-binding. |
   +-----------------------------------------------------+
EXPECTED_PATTERN[1013] in test/parser/fuzz.6.pytch, line 1, character 20:
Error: I was expecting a pattern after 'let'.
   +---------------------------------------------------------------------+
   | test/parser/fuzz.6.pytch                                            |
 1 | let!f/o = print@let                                                 |
   |                    ^ Error: I was expecting a pattern after 'let'.  |
   |                 ^~~ Note: This is the beginning of the let-binding. |
   +---------------------------------------------------------------------+
