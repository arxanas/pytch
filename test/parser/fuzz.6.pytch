let!f/o = print@let