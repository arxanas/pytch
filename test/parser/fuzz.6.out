SyntaxTree
    LetExpr
        Token 'let'
        <missing>
        <missing>
        <missing>
        <missing>
        Leading '!'
        Leading 'f'
        Leading '/'
        Leading 'o'
        Leading ' ='
        Leading ' print'
        Leading '@'
        Token DUMMY_IN ''
        LetExpr
            Token 'let'
            <missing>
            <missing>
            <missing>
            <missing>
            Token DUMMY_IN ''
            <missing>
    Token EOF ''
